### PR TITLE
AT_01.05.10 HeaderUser configure redirected

### PR DESCRIPTION
**Pull request by AlisaKapitanova**
https://trello.com/c/hl6m8cSj/906-at010510-headeruser-configure-redirected-to-the-configure-page

### CODE REVIEW

**Review by Julia-Cherniak — APPROVED** [review]

**Review by SvetlanaKazakova — APPROVED** [review]

File: cypress/e2e/headerUserConfigure.cy.js [hunk 1]
```diff
@@ -156,4 +161,9 @@ describe('Header User configure', () => {
         cy.get(`${saveButton}`).click();
         cy.get('#description-link').should('contain', 'Add description');
     });
+
+    it('AT_01.05.10 | Header>User configure redirected', () => {
+        cy.openConfigurationPage();
+        cy.get('.jenkins-form-label').eq(0).should('contain', 'Full Name')
```
Comment (Rom-T) [inline cypress/e2e/headerUserConfigure.cy.js:167]: Please move `Full Name` in the fixtures

Comment (AlisaKapitanova) [reply cypress/e2e/headerUserConfigure.cy.js:167]: What do you mean? Sorry, I do not understand. I did not use fixtures in my AT.

Comment (Rom-T) [reply cypress/e2e/headerUserConfigure.cy.js:167]: This does not apply to AT. We transfer the entire data to fixtures.

Comment (Rom-T) [reply cypress/e2e/headerUserConfigure.cy.js:167]: As an example: Look how they did it in the test AT_01.05_014

Comment (AlisaKapitanova) [reply cypress/e2e/headerUserConfigure.cy.js]: I do it. Thank you for review

**Pull request merged**